Sources/GRPC/AsyncAwaitSupport/GRPCAsyncResponseStreamWriter.swift

```diff
@@ -19,26 +19,90 @@
 /// Writer for server-streaming RPC handlers to provide responses.
 @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
 public struct GRPCAsyncResponseStreamWriter<Response: Sendable>: Sendable {
+  public struct RecordingAsyncSequence: AsyncSequence {
```

Comment:
I don't think 'recording' is right here: it doesn't keep all the requests it's seen. We should change this name (and elsewhere).